packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/error/NoPrefferedPackageFound.java

```diff
@@ -0,0 +1,10 @@
+package expo.modules.webbrowser.error;
+
+import org.unimodules.core.errors.CodedException;
+
+public class NoPrefferedPackageFound extends CodedException {
+
+  public NoPrefferedPackageFound() {
+    super("Cannot determine preferred package without satisfying it.");
```

Comment:
Is there anything a developer could do to get rid of this exception? Do you think it could be included in the exception message?

Reply:
Hmm... He could provide some package, but in fact, he shouldn't. This should be ignored, as this functionality is not critical and probably user doesn't hav a browser that supports it (hence developer cannot do anything) or he ha chosen as default browser some browser which do not support this case. In both cases developer should ignore this error, others seems to be marginal.